MDNetworking/Wrappers/MDListDataConverters.cs

```diff
@@ -21,10 +21,7 @@ public interface IMDDataConverter
         /// <param name="CurrentObject">The object that we are about to replace</param>
         /// <param name="Parameters">The parameters for conversion</param>
         /// <returns>The new object</returns>
-        object CovertBackToObject(object CurrentObject, object[] Parameters);
-
-        ///<Summary>Return how many parameters were used by the last ConvertFromObject call</summary>
-        int GetParametersConsumedByLastConversion();
+        object ConvertBackToObject(object CurrentObject, object[] Parameters);
```

Comment:
(line unrelated)
This file should probably be renamed since it's not List specific: MDDataConverters.cs